Update README.md (#9)
T-J-Summer/Final-Project: README.md

#### What do i plan to investigate?	
## Finding the best dataset	
#### What dataset am I looking at in order to find this?	

> I have decided to get my data from Big Query public datasets because I wanted to explore global data in order to explore my question and answer wether my hypothesis is correct. I also wanted to use Google BgQuery for this data because I have been using it for the past 9 weeks and have found that it is extremely up to date with some data being recorded from 2-3 days ago.

Reviewer: ```suggestion
I have decided to get my data from BigQuery public datasets because I wanted to test a hypothesis that I initially formulated: "the quality of education positively affects the economic and social success of a country." I have chosen to use SQL language to obtain and filter my data initial dataset as it would make my subsequent data exploration in Pandas faster. Moreover, BigQuery's SQL engine is powerful, which could also reduce the time I would spend exploring the initial data.
```

> Because of this i am choosing to look at the international data from BigQuery, i am going to use ‘internatinal_education’ because it has all the statistics regarding student population in each country, this dataset has all countries and the population of different school sectors (e.g. Secondary, Primary…). 

Reviewer: ```suggestion
I have chosen to look at the World Bank international education dataset via BigQuery. The dataset has the necessary statistics, e.g., the student population in each country, and covers all the countries in 2022 and the different school sectors, e.g., Secondary, Primary, and so on.
```

> internatinal_education - this dataset shows the education statistics for diffrent countries across the years. 

Reviewer: ```suggestion
The schema of the first data set I have used is screenshot below:
```

![education_schema](Education_schema.png)

> The second dataset that i am going to look at is from the world bank, i chose to source information from here because it is a reperitable source and can be trusted. This dataset will enable me to see which countries are classed as high, middle or low income countries this then will enable me to match this data with the education datasets and establish whether there is a correlation between a countries income and their education population. 

Reviewer: ```suggestion

```

> #### Why have i chose this data?	

Reviewer: ```suggestion
#### Why have I chosen this data?
	
```
